packages/server/worker/src/lib/consume/job-queue-worker.ts

```diff
@@ -46,7 +46,9 @@ export const jobQueueWorker = (log: FastifyBaseLogger) => ({
                 }
 
                 assertNotNullOrUndefined(jobId, 'jobId')
-                const { shouldRateLimit } = await workerJobRateLimiter(log).shouldBeLimited(jobId, job.data)
+                const maxConcurrentJobsPerProject = isNil(job.data.projectId) ? null : Number(await workerDistributedStore.get(getProjectMaxConcurrentJobsKey(job.data.projectId)))
```

Comment:
Move this to function and in case it null it should the default settings (environment)